hmmlearn/base.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, n_components=1, startprob=None, transmat=None,
         self.transmat_prior = transmat_prior
         self.algorithm = algorithm
         self.random_state = random_state
+        self.logprob_ = None
+        self.n_iterations_performed = None
+        self.logprob_delta = None
```

Comment:
I would think it would be a more compact model if we saved these three variables, rather than the full history of log probabilities. (For my usage case right now, we sometimes need 100k + iterations . And if we do ensemble methods, if each model is storing an array of 100k+ values, that is a huge consumption of memory that is not required). It would be much more efficient (memory wise) to simply store the final logprob, number of iterations performed, and the last delta achieved.